src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1508,6 +1524,11 @@ void TorrentHandle::handleSaveResumeDataAlert(libtorrent::save_resume_data_alert
         resumeData["qBt-magnetUri"] = toMagnetUri().toStdString();
         resumeData["qBt-paused"] = isPaused();
         resumeData["qBt-forced"] = isForced();
+        // Both firstLastPiecePriority and sequential need to be stored in the
+        // resume data if there is no metadata, otherwise they won't be
+        // restored if qBittorrent quites before the metadata is retrieved:
```

Comment:
I'm generally a descriptivist when it comes to language and I think at this point "data" and "metadata" can both be used as singular words as it's undeniably become a common usage. But regardless I'll make the change.